libbeat/docs/outputconfig.asciidoc

```diff
@@ -739,21 +739,39 @@ use in Logstash for indexing and filtering:
 for more about the `@metadata` field.
 <2> The default is {beat_default_index_prefix}. To change this value, set the
 <<logstash-index,`index`>> option in the {beatname_uc} config file.
-<3> The beats current version.
-<4> The value of `type` is currently hardcoded to `doc`. It was used by previous
-Logstash configs to set the type of the document in Elasticsearch.
-
+<3> The current version of {beatname_uc}.
+endif::[]
 
-WARNING: The `@metadata.type` field, added by the Logstash output, is
-deprecated, hardcoded to `doc`, and will be removed in {beatname_uc} 7.0.
+ifdef::apm-server[]
+["source","json",subs="attributes"]
+------------------------------------------------------------------------------
+{
+    ...
+    "@metadata": { <1>
+      "beat": "{beat_default_index_prefix}", <2>
+      "pipeline":"apm", <3>
+      "version": "{stack-version}" <4>
+    }
+}
+------------------------------------------------------------------------------
+<1> {beatname_uc} uses the `@metadata` field to send metadata to Logstash. See the
+{logstash-ref}/event-dependent-configuration.html#metadata[Logstash documentation]
+for more about the `@metadata` field.
+<2> The default is {beat_default_index_prefix}. To change this value, set the
+<<logstash-index,`index`>> option in the {beatname_uc} config file.
+<3> The default pipeline configuration: `apm`. Additional pipelines can be enabled
+with a {logstash-ref}/use-ingest-pipelines.html[Logstash pipeline config].
+<4> The current version of {beatname_uc}.
+endif::[]
 
 You can access this metadata from within the Logstash config file to set values
 dynamically based on the contents of the metadata.
 
-For example, the following Logstash configuration file for versions 2.x and
-5.x sets Logstash to use the index and document type reported by Beats for
-indexing events into Elasticsearch:
+For example, the following Logstash configuration file for version 7.x sets
```

Comment:
Ouch...that should have been updated before now. Maybe we should not give a version at all here.

Reply:
Good idea. Removed.